### DISCUSSION TITLE
Error in running 1_uniaxial example

### DISCUSSION THREAD

**Discussion by sushdash** [neperfepx/neper#302] [Q&A] [answered]
Hey! 
I am currently a beginner in Neper! I know its a very simple doubt! Even after following all the steps correctly, I am getting the attached error while running  the simulation module in neper after the fepx simulation. Please help me out with this.
 
![Screenshot (9)](https://user-images.githubusercontent.com/92076141/136327624-a9342960-ebb6-4efc-b8cb-23a5354e533e.png)

**Comment by smallrain1996** [comment]
If you want to use FEPX, you can refer to the video below or follow the FEPX user manual very carefully and you can check if there are any steps that have not been done properly.

https://www.youtube.com/watch?v=DwlhjnHtk3E&t=937s

**Comment by rquey** [comment, marked as answer]
You have to provide the name of the FEPX result directory as argument to the `neper -S` command:
```
$ neper -S name_of_the_fepx_result_directory
```
